supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigHashingStrategy.scala

```diff
@@ -9,22 +9,31 @@ import cromwell.backend.standard.callcaching.StandardFileHashingActor.SingleFile
 import cromwell.core.path.{Path, PathFactory}
 import cromwell.util.TryWithResource._
 import net.ceedubs.ficus.Ficus._
+import net.jpountz.xxhash.XXHashFactory
 import org.apache.commons.codec.digest.DigestUtils
 import org.slf4j.{Logger, LoggerFactory}
 
 import scala.util.{Failure, Try}
 
 object ConfigHashingStrategy {
   val logger: Logger = LoggerFactory.getLogger(getClass)
-  val defaultStrategy = HashFileStrategy(checkSiblingMd5 = false)
+  val defaultStrategy = HashFileMd5Strategy(checkSiblingMd5 = false)
 
   def apply(hashingConfig: Config): ConfigHashingStrategy = {
       val checkSiblingMd5 = hashingConfig.as[Option[Boolean]]("check-sibling-md5").getOrElse(false)
 
+      // Fingerprint strategy by default checks the first 10 MB (10485760 bytes) for performance reasons.
+      // 100 MB will take to much time on network file systems. 1 MB might not be unique enough.
+      // The value is user configurable.
+      lazy val fingerprintSize = hashingConfig.as[Option[Long]]("fingerprint-size").getOrElse(10485760L)
```

Comment:
```suggestion
      lazy val fingerprintSize = hashingConfig.as[Option[Long]]("fingerprint-size").getOrElse(10 * (1 << 20))
```

Reply:
I will do 10 * 1024 * 1024 here as it is more clear than either 10485670 or `10 * (1 <<20)`